packages/react/src/ActionMenu/ActionMenu.tsx

```diff
@@ -165,7 +176,8 @@ const Overlay: React.FC<React.PropsWithChildren<MenuOverlayProps>> = ({
           value={{
             container: 'ActionMenu',
             listRole: 'menu',
-            listLabelledBy: ariaLabelledby || anchorId,
+            // If there is a custom aria-labelledby, use that. Otherwise, use the tooltipId if exist. If not, use anchor id.
+            listLabelledBy: ariaLabelledby || tooltipId || anchorId,
```

Comment:
(non blocking) instead of using `tooltipId` variable on this line, can we use `anchorAriaLabelledby`?

it feels better saying that the list is labelled by `list aria-labelledby` or the "anchor label", which can be either `anchor aria-labelledby` or just `anchorId`